Cannot find module ...

## Bug Report
Note that, the bug you're reporting may have registered in the [issues](https://github.com/samchon/nestia/search?type=issues) by another user. Even the bug you're reporting may have been fixed in the `@next` version. In such reasons, I recommend you to check the old [issues](https://github.com/samchon/nestia/search?type=issues) and reproduct your code with the `@next` version before publishing the bug reporting issue.

```bash
`npm install --save-dev nestia@next`
```

When the same error occurs even in the `@next` version, then please fill the below template:

### Summary
Write a short summary of the bug in here.

  - **SDK Version**: ^3.0.10-dev.20220725
  - **Expected behavior**: 
  - **Actual behavior**: 

Running the command results in an MODULE NOT FOUND error. 

### Code occuring the bug
```typescript
/* Demonstration code occuring the bug you're reporting */
```

Write the demonstration code occuring the bug you're reporting.

However, as the `nestia` is not a typical library who being used by the `import` statement but a executable program generating the SDK (Software Development Kit), it would be difficult to describe the bug with only the demonstration code.

In that case, you'd better to make a demonstration project occuring the bug to report. After the demonstration repository composition, fill the below `bash` script section. Of course, if you can describe the bug clearly, without the demonstration project construction, just skip it. Creating the demonstration repository is not essential.

```bash
npx nestia sdk
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'typescript-json/lib/factories/CommentFactory'
Require stack:
- /nest-api/node_modules/nestia/lib/generates/SwaggerGenerator.js
- /nest-api/node_modules/nestia/lib/NestiaApplication.js
- /nest-api/node_modules/nestia/lib/executable/internal/NestiaCommand.js
- /nest-api/node_modules/nestia/lib/executable/nestia.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/nest-api/node_modules/nestia/lib/generates/SwaggerGenerator.js:20:26)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/nest-api/node_modules/nestia/lib/generates/SwaggerGenerator.js',
    '/nest-api/node_modules/nestia/lib/NestiaApplication.js',
    '/nest-api/node_modules/nestia/lib/executable/internal/NestiaCommand.js',
    '/nest-api/node_modules/nestia/lib/executable/nestia.js'
  ]
}
```

--- a/demo/encrypted/swagger.json
+++ b/demo/encrypted/swagger.json
@@ -150,7 +150,8 @@
           "name",
           "mobile",
           "company"
-        ]
+        ],
+        "jsDocTags": []
       },
       "ISeller": {
         "type": "object",
@@ -195,7 +196,18 @@
           "company",
           "created_at"
         ],
-        "description": "Seller information."
+        "description": "Seller information.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ISeller.ILogin": {
         "type": "object",
@@ -213,7 +225,8 @@
         "required": [
           "email",
           "password"
-        ]
+        ],
+        "jsDocTags": []
       },
       "ISeller.IChangePassword": {
         "type": "object",
@@ -231,7 +244,8 @@
         "required": [
           "old_password",
           "new_password"
-        ]
+        ],
+        "jsDocTags": []
       }
     }
   },

--- a/demo/generic/swagger.json
+++ b/demo/generic/swagger.json
@@ -308,7 +308,8 @@
           "extension",
           "files"
         ],
-        "description": "Store info."
+        "description": "Store info.",
+        "jsDocTags": []
       },
       "IAttachmentFile": {
         "type": "object",
@@ -335,7 +336,18 @@
           "extension",
           "url"
         ],
-        "description": "Attachemd (physical) file info."
+        "description": "Attachemd (physical) file info.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ISaleInquiry_lt_ISaleArticle.IContent_gt_": {
         "type": "object",
@@ -383,7 +395,35 @@
           "contents",
           "created_at"
         ],
-        "description": "Inquiry article.\n\nSub-type of article and super-type of question and answer."
+        "description": "Inquiry article.\n\nSub-type of article and super-type of question and answer.",
+        "jsDocTags": [
+          {
+            "name": "template",
+            "text": [
+              {
+                "text": "Content",
+                "kind": "typeParameterName"
+              },
+              {
+                "text": " ",
+                "kind": "space"
+              },
+              {
+                "text": "Content type",
+                "kind": "text"
+              }
+            ]
+          },
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ISaleArticle_lt_ISaleArticle.IContent_gt_.Nullable": {
         "type": "object",
@@ -420,7 +460,35 @@
           "contents",
           "created_at"
         ],
-        "description": "Article about a sale.\n\nThe `ISaleArticle` is a super type interface. Many sub-type articles would be\ndesigned by extending this super type interface `ISaleArticle`."
+        "description": "Article about a sale.\n\nThe `ISaleArticle` is a super type interface. Many sub-type articles would be\ndesigned by extending this super type interface `ISaleArticle`.",
+        "jsDocTags": [
+          {
+            "name": "template",
+            "text": [
+              {
+                "text": "Content",
+                "kind": "typeParameterName"
+              },
+              {
+                "text": " ",
+                "kind": "space"
+              },
+              {
+                "text": "Content type.",
+                "kind": "text"
+              }
+            ]
+          },
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ISaleArticle.IContent": {
         "type": "object",
@@ -474,7 +542,8 @@
           "extension",
           "files"
         ],
-        "description": "Content info."
+        "description": "Content info.",
+        "jsDocTags": []
       },
       "ISaleReview.IStore": {
         "type": "object",
@@ -522,7 +591,8 @@
           "extension",
           "files"
         ],
-        "description": "Store info of the review."
+        "description": "Store info of the review.",
+        "jsDocTags": []
       },
       "ISaleInquiry_lt_ISaleReview.IContent_gt_": {
         "type": "object",
@@ -570,7 +640,35 @@
           "contents",
           "created_at"
         ],
-        "description": "Inquiry article.\n\nSub-type of article and super-type of question and answer."
+        "description": "Inquiry article.\n\nSub-type of article and super-type of question and answer.",
+        "jsDocTags": [
+          {
+            "name": "template",
+            "text": [
+              {
+                "text": "Content",
+                "kind": "typeParameterName"
+              },
+              {
+                "text": " ",
+                "kind": "space"
+              },
+              {
+                "text": "Content type",
+                "kind": "text"
+              }
+            ]
+          },
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ISaleReview.IContent": {
         "type": "object",
@@ -630,7 +728,8 @@
           "extension",
           "files"
         ],
-        "description": "Content info of the review."
+        "description": "Content info of the review.",
+        "jsDocTags": []
       }
     }
   }

--- a/demo/multiple-paths/swagger.json
+++ b/demo/multiple-paths/swagger.json
@@ -238,7 +238,8 @@
           "extension",
           "annonymous"
         ],
-        "description": "Store info."
+        "description": "Store info.",
+        "jsDocTags": []
       },
       "ISaleArticleComment": {
         "type": "object",
@@ -291,7 +292,18 @@
           "contents",
           "created_at"
         ],
-        "description": "Comment wrote on a sale related article.\n\nWhen an article of a sale has been enrolled, all of the participants like consumers and\nsellers can write a comment on that article. However, when the writer is a consumer, the\nconsumer can hide its name through the annoymous option.\n\nAlso, writing a reply comment for a specific comment is possible and in that case, the\n{@link ISaleArticleComment.parent_id} property would be activated."
+        "description": "Comment wrote on a sale related article.\n\nWhen an article of a sale has been enrolled, all of the participants like consumers and\nsellers can write a comment on that article. However, when the writer is a consumer, the\nconsumer can hide its name through the annoymous option.\n\nAlso, writing a reply comment for a specific comment is possible and in that case, the\n{@link ISaleArticleComment.parent_id} property would be activated.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ISaleArticleComment.IContent": {
         "type": "object",
@@ -318,7 +330,8 @@
           "body",
           "created_at"
         ],
-        "description": "Content info."
+        "description": "Content info.",
+        "jsDocTags": []
       }
     }
   }

--- a/demo/recursive/swagger.json
+++ b/demo/recursive/swagger.json
@@ -132,7 +132,18 @@
           "code",
           "name"
         ],
-        "description": "Systematic category info.\n\nThis `ICategory` has been designed to represent a systematic classfication. Also, its\nclassification has hierarchical and recursive relationship. Therefore, its structure has\na tree shape through the {@link ICategory.children} property.\n\nIf you want to see invert structure for getting parent information repeatedly, use the\n{@link ICategory.IInvert} type instead."
+        "description": "Systematic category info.\n\nThis `ICategory` has been designed to represent a systematic classfication. Also, its\nclassification has hierarchical and recursive relationship. Therefore, its structure has\na tree shape through the {@link ICategory.children} property.\n\nIf you want to see invert structure for getting parent information repeatedly, use the\n{@link ICategory.IInvert} type instead.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ICategory.IInvert": {
         "type": "object",
@@ -164,7 +175,8 @@
           "code",
           "name"
         ],
-        "description": "Invert structure of the {@link ICategory}."
+        "description": "Invert structure of the {@link ICategory}.",
+        "jsDocTags": []
       },
       "ICategory.IInvert.Nullable": {
         "type": "object",
@@ -196,7 +208,8 @@
           "code",
           "name"
         ],
-        "description": "Invert structure of the {@link ICategory}."
+        "description": "Invert structure of the {@link ICategory}.",
+        "jsDocTags": []
       }
     }
   }

--- a/demo/safe/swagger.json
+++ b/demo/safe/swagger.json
@@ -117,7 +117,8 @@
           "extension",
           "annonymous"
         ],
-        "description": "Store info."
+        "description": "Store info.",
+        "jsDocTags": []
       },
       "ISaleArticleComment": {
         "type": "object",
@@ -170,7 +171,18 @@
           "contents",
           "created_at"
         ],
-        "description": "Comment wrote on a sale related article.\n\nWhen an article of a sale has been enrolled, all of the participants like consumers and\nsellers can write a comment on that article. However, when the writer is a consumer, the\nconsumer can hide its name through the annoymous option.\n\nAlso, writing a reply comment for a specific comment is possible and in that case, the\n{@link ISaleArticleComment.parent_id} property would be activated."
+        "description": "Comment wrote on a sale related article.\n\nWhen an article of a sale has been enrolled, all of the participants like consumers and\nsellers can write a comment on that article. However, when the writer is a consumer, the\nconsumer can hide its name through the annoymous option.\n\nAlso, writing a reply comment for a specific comment is possible and in that case, the\n{@link ISaleArticleComment.parent_id} property would be activated.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ISaleArticleComment.IContent": {
         "type": "object",
@@ -197,7 +209,8 @@
           "body",
           "created_at"
         ],
-        "description": "Content info."
+        "description": "Content info.",
+        "jsDocTags": []
       }
     }
   }

--- a/demo/union/swagger.json
+++ b/demo/union/swagger.json
@@ -134,10 +134,10 @@
                 "schema": {
                   "oneOf": [
                     {
-                      "$ref": "#/components/schemas/ISaleInquiry_lt_ISaleReview.IContent_gt_"
+                      "$ref": "#/components/schemas/ISaleInquiry_lt_ISaleArticle.IContent_gt_"
                     },
                     {
-                      "$ref": "#/components/schemas/ISaleInquiry_lt_ISaleArticle.IContent_gt_"
+                      "$ref": "#/components/schemas/ISaleInquiry_lt_ISaleReview.IContent_gt_"
                     }
                   ]
                 }
@@ -298,7 +298,8 @@
         "required": [
           "code",
           "referrer"
-        ]
+        ],
+        "jsDocTags": []
       },
       "ISaleInquiry.IRequest": {
         "type": "object",
@@ -316,7 +317,18 @@
           "limit": {
             "type": "number",
             "nullable": false,
-            "description": "Maximum number of records per a page."
+            "description": "Maximum number of records per a page.",
+            "jsDocTags": [
+              {
+                "name": "default",
+                "text": [
+                  {
+                    "text": "100",
+                    "kind": "text"
+                  }
+                ]
+              }
+            ]
           },
           "search_fields": {
             "type": "array",
@@ -345,7 +357,8 @@
           }
         },
         "nullable": false,
-        "description": "Page request info."
+        "description": "Page request info.",
+        "jsDocTags": []
       },
       "IPage_lt_ISaleEntireArtcle.ISummary_gt_": {
         "type": "object",
@@ -366,7 +379,8 @@
                   "$ref": "#/components/schemas/ISaleReview.ISummary",
                   "description": "List of the records."
                 }
-              ]
+              ],
+              "description": "List of the records."
             },
             "nullable": false,
             "description": "List of the records."
@@ -377,7 +391,18 @@
           "pagination",
           "data"
         ],
-        "description": "Paginated records."
+        "description": "Paginated records.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "IPage.IPagination": {
         "type": "object",
@@ -390,7 +415,18 @@
           "limit": {
             "type": "number",
             "nullable": false,
-            "description": "Maximum number of records per a page."
+            "description": "Maximum number of records per a page.",
+            "jsDocTags": [
+              {
+                "name": "default",
+                "text": [
+                  {
+                    "text": "100",
+                    "kind": "text"
+                  }
+                ]
+              }
+            ]
           },
           "total_count": {
             "type": "number",
@@ -410,7 +446,8 @@
           "total_count",
           "total_pages"
         ],
-        "description": "Pagenation info."
+        "description": "Pagenation info.",
+        "jsDocTags": []
       },
       "ISaleInquiry.ISummary": {
         "type": "object",
@@ -461,7 +498,8 @@
           "created_at",
           "updated_at"
         ],
-        "description": "Summarized info of the inquiry article."
+        "description": "Summarized info of the inquiry article.",
+        "jsDocTags": []
       },
       "ISaleReview.ISummary": {
         "type": "object",
@@ -518,7 +556,237 @@
           "created_at",
           "updated_at"
         ],
-        "description": "Summarized info of the review."
+        "description": "Summarized info of the review.",
+        "jsDocTags": []
+      },
+      "ISaleInquiry_lt_ISaleArticle.IContent_gt_": {
+        "type": "object",
+        "properties": {
+          "writer": {
+            "type": "string",
+            "nullable": false,
+            "description": "Name of the writer."
+          },
+          "answer": {
+            "$ref": "#/components/schemas/ISaleArticle_lt_ISaleArticle.IContent_gt_.Nullable",
+            "description": "Formal answer from the seller."
+          },
+          "id": {
+            "type": "number",
+            "nullable": false,
+            "description": "Primary Key."
+          },
+          "hit": {
+            "type": "number",
+            "nullable": false,
+            "description": "Hit count.\n\nHow many consumers had read."
+          },
+          "contents": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/ISaleArticle.IContent",
+              "description": "List of contents.\n\nWhen the article writer tries to modify content, it would not modify the article\ncontent but would be accumulated. Therefore, all of the people can read how\nthe content has been changed."
+            },
+            "nullable": false,
+            "description": "List of contents.\n\nWhen the article writer tries to modify content, it would not modify the article\ncontent but would be accumulated. Therefore, all of the people can read how\nthe content has been changed."
+          },
+          "created_at": {
+            "type": "string",
+            "nullable": false,
+            "description": "Creation time."
+          }
+        },
+        "nullable": false,
+        "required": [
+          "writer",
+          "answer",
+          "id",
+          "hit",
+          "contents",
+          "created_at"
+        ],
+        "description": "Inquiry article.\n\nSub-type of article and super-type of question and answer.",
+        "jsDocTags": [
+          {
+            "name": "template",
+            "text": [
+              {
+                "text": "Content",
+                "kind": "typeParameterName"
+              },
+              {
+                "text": " ",
+                "kind": "space"
+              },
+              {
+                "text": "Content type",
+                "kind": "text"
+              }
+            ]
+          },
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
+      },
+      "ISaleArticle_lt_ISaleArticle.IContent_gt_.Nullable": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "number",
+            "nullable": false,
+            "description": "Primary Key."
+          },
+          "hit": {
+            "type": "number",
+            "nullable": false,
+            "description": "Hit count.\n\nHow many consumers had read."
+          },
+          "contents": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/ISaleArticle.IContent",
+              "description": "List of contents.\n\nWhen the article writer tries to modify content, it would not modify the article\ncontent but would be accumulated. Therefore, all of the people can read how\nthe content has been changed."
+            },
+            "nullable": false,
+            "description": "List of contents.\n\nWhen the article writer tries to modify content, it would not modify the article\ncontent but would be accumulated. Therefore, all of the people can read how\nthe content has been changed."
+          },
+          "created_at": {
+            "type": "string",
+            "nullable": false,
+            "description": "Creation time."
+          }
+        },
+        "nullable": true,
+        "required": [
+          "id",
+          "hit",
+          "contents",
+          "created_at"
+        ],
+        "description": "Article about a sale.\n\nThe `ISaleArticle` is a super type interface. Many sub-type articles would be\ndesigned by extending this super type interface `ISaleArticle`.",
+        "jsDocTags": [
+          {
+            "name": "template",
+            "text": [
+              {
+                "text": "Content",
+                "kind": "typeParameterName"
+              },
+              {
+                "text": " ",
+                "kind": "space"
+              },
+              {
+                "text": "Content type.",
+                "kind": "text"
+              }
+            ]
+          },
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
+      },
+      "ISaleArticle.IContent": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "nullable": false,
+            "description": "Primary Key"
+          },
+          "title": {
+            "type": "string",
+            "nullable": false,
+            "description": "Title of the content."
+          },
+          "body": {
+            "type": "string",
+            "nullable": false,
+            "description": "Body of the content."
+          },
+          "files": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/IAttachmentFile",
+              "description": "Attached files."
+            },
+            "nullable": false,
+            "description": "Attached files."
+          },
+          "created_at": {
+            "type": "string",
+            "nullable": false,
+            "description": "Creation time."
+          }
+        },
+        "nullable": false,
+        "required": [
+          "id",
+          "title",
+          "body",
+          "files",
+          "created_at"
+        ],
+        "description": "Content info.",
+        "jsDocTags": []
+      },
+      "IAttachmentFile": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "number",
+            "nullable": false,
+            "description": "Primary Key."
+          },
+          "name": {
+            "type": "string",
+            "nullable": false,
+            "description": "Name of the file."
+          },
+          "extension": {
+            "type": "string",
+            "nullable": true,
+            "description": "Extension of the file, but ommitable."
+          },
+          "url": {
+            "type": "string",
+            "nullable": false,
+            "description": "Full URL path of the file."
+          }
+        },
+        "nullable": false,
+        "required": [
+          "id",
+          "name",
+          "extension",
+          "url"
+        ],
+        "description": "Attachemd (physical) file info.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "ISaleInquiry_lt_ISaleReview.IContent_gt_": {
         "type": "object",
@@ -566,120 +834,35 @@
           "contents",
           "created_at"
         ],
-        "description": "Inquiry article.\n\nSub-type of article and super-type of question and answer."
-      },
-      "ISaleArticle_lt_ISaleArticle.IContent_gt_.Nullable": {
-        "type": "object",
-        "properties": {
-          "id": {
-            "type": "number",
-            "nullable": false,
-            "description": "Primary Key."
+        "description": "Inquiry article.\n\nSub-type of article and super-type of question and answer.",
+        "jsDocTags": [
+          {
+            "name": "template",
+            "text": [
+              {
+                "text": "Content",
+                "kind": "typeParameterName"
+              },
+              {
+                "text": " ",
+                "kind": "space"
+              },
+              {
+                "text": "Content type",
+                "kind": "text"
+              }
+            ]
           },
-          "hit": {
-            "type": "number",
-            "nullable": false,
-            "description": "Hit count.\n\nHow many consumers had read."
-          },
-          "contents": {
-            "type": "array",
-            "items": {
-              "$ref": "#/components/schemas/ISaleArticle.IContent",
-              "description": "List of contents.\n\nWhen the article writer tries to modify content, it would not modify the article\ncontent but would be accumulated. Therefore, all of the people can read how\nthe content has been changed."
-            },
-            "nullable": false,
-            "description": "List of contents.\n\nWhen the article writer tries to modify content, it would not modify the article\ncontent but would be accumulated. Therefore, all of the people can read how\nthe content has been changed."
-          },
-          "created_at": {
-            "type": "string",
-            "nullable": false,
-            "description": "Creation time."
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
           }
-        },
-        "nullable": true,
-        "required": [
-          "id",
-          "hit",
-          "contents",
-          "created_at"
-        ],
-        "description": "Article about a sale.\n\nThe `ISaleArticle` is a super type interface. Many sub-type articles would be\ndesigned by extending this super type interface `ISaleArticle`."
-      },
-      "ISaleArticle.IContent": {
-        "type": "object",
-        "properties": {
-          "id": {
-            "type": "string",
-            "nullable": false,
-            "description": "Primary Key"
-          },
-          "title": {
-            "type": "string",
-            "nullable": false,
-            "description": "Title of the content."
-          },
-          "body": {
-            "type": "string",
-            "nullable": false,
-            "description": "Body of the content."
-          },
-          "files": {
-            "type": "array",
-            "items": {
-              "$ref": "#/components/schemas/IAttachmentFile",
-              "description": "Attached files."
-            },
-            "nullable": false,
-            "description": "Attached files."
-          },
-          "created_at": {
-            "type": "string",
-            "nullable": false,
-            "description": "Creation time."
-          }
-        },
-        "nullable": false,
-        "required": [
-          "id",
-          "title",
-          "body",
-          "files",
-          "created_at"
-        ],
-        "description": "Content info."
-      },
-      "IAttachmentFile": {
-        "type": "object",
-        "properties": {
-          "id": {
-            "type": "number",
-            "nullable": false,
-            "description": "Primary Key."
-          },
-          "name": {
-            "type": "string",
-            "nullable": false,
-            "description": "Name of the file."
-          },
-          "extension": {
-            "type": "string",
-            "nullable": true,
-            "description": "Extension of the file, but ommitable."
-          },
-          "url": {
-            "type": "string",
-            "nullable": false,
-            "description": "Full URL path of the file."
-          }
-        },
-        "nullable": false,
-        "required": [
-          "id",
-          "name",
-          "extension",
-          "url"
-        ],
-        "description": "Attachemd (physical) file info."
+        ]
       },
       "ISaleReview.IContent": {
         "type": "object",
@@ -728,55 +911,8 @@
           "files",
           "created_at"
         ],
-        "description": "Content info of the review."
-      },
-      "ISaleInquiry_lt_ISaleArticle.IContent_gt_": {
-        "type": "object",
-        "properties": {
-          "writer": {
-            "type": "string",
-            "nullable": false,
-            "description": "Name of the writer."
-          },
-          "answer": {
-            "$ref": "#/components/schemas/ISaleArticle_lt_ISaleArticle.IContent_gt_.Nullable",
-            "description": "Formal answer from the seller."
-          },
-          "id": {
-            "type": "number",
-            "nullable": false,
-            "description": "Primary Key."
-          },
-          "hit": {
-            "type": "number",
-            "nullable": false,
-            "description": "Hit count.\n\nHow many consumers had read."
-          },
-          "contents": {
-            "type": "array",
-            "items": {
-              "$ref": "#/components/schemas/ISaleArticle.IContent",
-              "description": "List of contents.\n\nWhen the article writer tries to modify content, it would not modify the article\ncontent but would be accumulated. Therefore, all of the people can read how\nthe content has been changed."
-            },
-            "nullable": false,
-            "description": "List of contents.\n\nWhen the article writer tries to modify content, it would not modify the article\ncontent but would be accumulated. Therefore, all of the people can read how\nthe content has been changed."
-          },
-          "created_at": {
-            "type": "string",
-            "nullable": false,
-            "description": "Creation time."
-          }
-        },
-        "nullable": false,
-        "required": [
-          "writer",
-          "answer",
-          "id",
-          "hit",
-          "contents",
-          "created_at"
-        ],
-        "description": "Inquiry article.\n\nSub-type of article and super-type of question and answer."
+        "description": "Content info of the review.",
+        "jsDocTags": []
       },
       "IPage_lt_ISaleInquiry.ISummary_gt_": {
         "type": "object",
@@ -800,7 +936,18 @@
           "pagination",
           "data"
         ],
-        "description": "Paginated records."
+        "description": "Paginated records.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       },
       "IPage_lt_ISaleReview.ISummary_gt_": {
         "type": "object",
@@ -824,7 +971,18 @@
           "pagination",
           "data"
         ],
-        "description": "Paginated records."
+        "description": "Paginated records.",
+        "jsDocTags": [
+          {
+            "name": "author",
+            "text": [
+              {
+                "text": "Jeongho Nam - https://github.com/samchon",
+                "kind": "text"
+              }
+            ]
+          }
+        ]
       }
     }
   }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "nestia",
-  "version": "3.0.13",
+  "version": "3.0.15",
   "description": "Automatic SDK and Document generator for the NestJS",
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
@@ -42,7 +42,8 @@
     "tgrid": "^0.8.7",
     "tsconfck": "^2.0.1",
     "tsconfig-paths": "^4.0.0",
-    "tstl": "^2.5.8"
+    "tstl": "^2.5.8",
+    "typescript-json": "^3.3.0"
   },
   "devDependencies": {
     "@types/cli": "^0.11.19",
@@ -59,7 +60,6 @@
     "ts-node": "10.8.x",
     "ttypescript": "^1.5.13",
     "typescript": "^4.7.4",
-    "typescript-json": "^3.1.1",
     "typescript-transform-paths": "^3.3.1",
     "uuid": "^8.3.2"
   }

